examples/DataApi-Voting-poll/template/modules/shared_data/src/main/scala/com/my/currency/shared_data/calculated_state/CalculatedState.scala

```diff
@@ -0,0 +1,51 @@
+package com.my.currency.shared_data.calculated_state
+
+import cats.Applicative
+import cats.effect.kernel.Async
+import cats.implicits.{catsSyntaxApplicativeId, toFunctorOps}
+import com.my.currency.shared_data.types.Types.VoteCalculatedState
+import eu.timepit.refined.types.numeric.NonNegLong
+import io.circe.syntax.EncoderOps
+import org.tessellation.schema.SnapshotOrdinal
+import org.tessellation.security.hash.Hash
+
+import java.security.MessageDigest
+import java.util.concurrent.atomic.AtomicReference
+
+object CalculatedState {
+
+  private val maybeVoteCalculatedState: AtomicReference[(SnapshotOrdinal, VoteCalculatedState)] = new AtomicReference(
```

Comment:
BTW, I missed this before:
The `maybe` prefix is commonly used to signify an `Option`, which this is not. If it were me I'd simply call this `stateRef`.

Reply:
Got it, I'll update